lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -249,15 +249,17 @@ class ReportUIFeatures {
       });
 
       this._dom.find('label', filterTemplate).setAttribute('for', id);
-      this._dom.find('.lh-3p-filter-count', filterTemplate).textContent =
+      this._dom.find('.lh-3p-filter__count', filterTemplate).textContent =
           `${thirdPartyRows.size}`;
-      this._dom.find('.lh-3p-ui-string', filterTemplate).textContent =
-          Util.UIStrings.thirdPartyResourcesLabel;
+      this._dom.find('.lh-3p-filter__show .lh-ui-string', filterTemplate).textContent =
+          Util.UIStrings.thirdPartyResourcesShowLabel;
+      this._dom.find('.lh-3p-filter__exclude .lh-ui-string', filterTemplate).textContent =
+          Util.UIStrings.thirdPartyResourcesExcludeLabel;
 
       // If all or none of the rows are 3rd party, disable the checkbox.
       if (thirdPartyRows.size === urlItems.length || !thirdPartyRows.size) {
         filterInput.disabled = true;
-        filterInput.checked = thirdPartyRows.size === urlItems.length;
+        filterInput.checked = thirdPartyRows.size !== urlItems.length;
```

Comment:
was this on purpose?

Reply:
yes. It was wrong before. and even more apparent now with the new styles

Reply:
(this is just a style thing. the value of the input doesnt matter at this point, but for css).

Reply:
gotcha, can we add some commentary and/or change the boolean?

is the commentary...

```
// The filter input should be checked when we are showing third party resources.
// We show third party resources when there are resources to be showing. 
filterInput.checked = thirdPartyRows.size > 0;
```

I think this is the opposite of your change though...

Reply:
Ah! that clears things up then :) let's annotate it as such then?

Reply:
hang on, I'm still not happy here. :)

shouldn't this just be `filterInput.checked = true;` in all cases? in no case does it make sense to have "Exclude third party (n)" be highlighted

Reply:
yup, done